apps/web/src/api/v2/util.ts

```diff
@@ -92,6 +118,7 @@ export const createBuild = async (params: {
       name: buildName,
       prNumber: params.prNumber ?? null,
       prHeadCommit: params.prHeadCommit ?? null,
+      pullRequestId: pullRequest?.id ? String(pullRequest.id) : null,
```

Comment:
If you need a cast here, there is a problem somewhere.

Reply:
I can't tell. There is number field for foreign keys that are typed has "string" in model, all over the project. I have no idea what should I do clean this mess

Reply:
I'll remove the cast and go along to satisfy the linter